src/main/java/bookstore/controller/BookController.java

```diff
@@ -26,30 +31,40 @@ public class BookController {
 
     @ResponseStatus(HttpStatus.OK)
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books",
+            description = "Get a list of all books")
+    public List<BookDto> getAll(@PageableDefault(page = 0, size = 2) Pageable pageable) {
```

Comment:
u can also add direction of sorting into parameters by default:
`direction = Sort.Direction.ASC`